Gemfile.lock

```diff
@@ -76,7 +76,7 @@ DEPENDENCIES
   kramdown-parser-gfm
 
 RUBY VERSION
-   ruby 2.7.0p0
+   ruby 2.6.3p62
 
 BUNDLED WITH
-   2.0.2
```

Comment:
Don't make changes to `Gemfile.lock`

Reply:
can you please share the location of the .DS_Store file. I just deleted two of them.

Reply:
found them. deleted them. please review it. thank you.